src/main/java/randoop/generation/ComponentManager.java

```diff
@@ -171,9 +149,12 @@ SIList<Sequence> getSequencesForType(Type cls) {
   }
 
   /**
-   * Returns component sequences that create values of the type required by the i-th input value of
-   * a statement that invokes the given operation. Also includes any applicable class- or
-   * package-level literals.
+   * Returns candidate sequences for the {@code i}-th input of {@code operation}: pool sequences
+   * that produce the required type, followed by literal sequences for the declaring class if
```

Comment:
"for the declaring class":  is this actually for the scope, or is it always just for the class?  If the latter, please explain why.